### PR TITLE
Fix (?) the rebuild

### PR DESCRIPTION
**Pull request by pkgw**
I think this should fix the rebuild PR.

Closes #72.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -19,11 +19,11 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make
+    - make            # [unix]
 
   host:
     - bzip2           # [unix]
-    - curl            # [unix]
+    - curl 7.61.1           # [unix]
```
Comment (jakirkham) [inline recipe/meta.yaml]: Is this still needed?

cc @marcelotrevisani

Comment (marcelotrevisani) [reply recipe/meta.yaml]: Nope, it was just a test

Comment (pkgw) [reply recipe/meta.yaml]: Given how long the CI is taking, and how low this is lying in the dependency stack, I'm tempted to leave this as-is ... but I can definitely take it out if folks would prefer.

Comment (mariusvniekerk) [reply recipe/meta.yaml]: I also think that given that cmake is basically always a `build` time dependency things like exact consistency with the conda-forge curl matter less?

**Comment by pkgw** [comment]
@conda-forge/cmake CI passed — I tend to think the curl version pin discussed above isn't a big deal, CI is slow lately, and this package seems to be blocking a lot of rebuilds, so maybe just merge as-is?

**Comment by jakirkham** [comment]
FYI the Linux build did not run on `master`. So new compiler packages of `cmake` did not deploy on Linux.

Have fixed that in issue ( https://github.com/conda-forge/cmake-feedstock/issues/74 ) and triggered a new CircleCI build, which is now queued. Hopefully will deploy soon.

**Pull request merged**